src/components/Modal/modal.scss

```diff
@@ -0,0 +1,7 @@
+@import "~vanilla-framework/scss/settings";
+
```

Comment:
Could you also add this file to `src/index.scss` for those that want to include all the css?